npm/rake-dist.js

```diff
@@ -0,0 +1,51 @@
+var child_process = require('child_process');
+var fs = require('fs');
+
+var Log = require('./log.js');
+var Builder = require('./builder.js');
+var builder = new Builder();
+var log = new Log();
+
+if (!fs.existsSync('build')) {
+  fs.mkdirSync('build');
+}
+
+var opalCompile = function(require, output) {
+  var stdout = child_process.execSync('opal --compile -I lib -I asciidoctor -g asciidoctor -r ' + require + ' --dynamic-require ignore --no-opal --no-exit > build/' + output);
+  process.stdout.write(stdout);
+}
+
+var opalCompileExtensions = function(names) {
```

Comment:
In your example `arguments` is a JSON object so `forEach` is undefined but I can write:

```javascript
var opalCompileExtensions = function(names) {
  names.forEach(opalCompileExtension);
}
```